src/org/thoughtcrime/securesms/ConversationAdapter.java

```diff
@@ -170,6 +170,14 @@ public long getItemId(int position) {
     return fromDb;
   }
 
+  public int msgIdToPosition(int msgId) {
```

Comment:
```suggestion
  /**
   * Returns the position of the message with msgId in the chat list, counted from the top
   */
  public int msgIdToPosition(int msgId) {
```
because it took me a while to understand what this does.